scaffolds/javascript/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  "parserOptions": {
```

Comment:
According to line 18 of this file (`'quotes': [1, 'single']`), the use of quotes should be limited to the single quotes. BTW, the JSON specification states that all strings (and hence, names) should use double quotes.

Reply:
Why use single quotes everywhere but this line? Just a small detail...

Reply:
Ah, right. I'd missed that. 

I added the file to the linter targets to avoid this in the futur :).

FYI, that config file is *not* JSON; it is plain JS -- JSON spec does not apply.

Reply:
OK, my mistake.